src/app/COMPONENTS/task/task.component.ts

```diff
@@ -26,16 +27,28 @@ import { MaterialModule } from '../../material.module';
 
 
 export class TaskComponent {
-  tasks$ = this.taskService.availableTasks$;
+  availableTasks$ = new BehaviorSubject<Task[]>([])
 
-  constructor(private taskService: TasksService) { }
+  constructor(private taskService: TasksService) {
+  }
+  ngOnInit() {
+    this.taskService.getAllTasks().subscribe(data => {
```

Comment:
This is a very awkward approach:
1. observable should only have $ at the end of variable names
2. You dont need to subscribe to the getAllTasks method, instead you could create an observable variable, which will receive the result from getAllTasks and then with async pipe show the result on screen